resources/HashingUtilities.ts

```diff
@@ -0,0 +1,26 @@
+const bcrypt = require('bcrypt');
+require('dotenv').config();
+
+
+export class HashingUtilities {
+
+    private saltRounds: number = this.getSaltFromEnv();
+    
+    private getSaltFromEnv(): number {
```

Comment:
Why is this called getSaltFromEnv ?

Reply:
Because it gets the salt value from env or uses the default one in case that it's missing.